src/backends/sentry_backend_crashpad.cpp

```diff
@@ -84,19 +84,27 @@ sentry__crashpad_backend_startup(sentry_backend_t *backend)
         }
     }
 
-    if (!handler_path || !sentry__path_is_file(handler_path)) {
+    // Starting the crashpad client needs either an absolute path, or an
+    // executable which can be resolved via $PATH, or otherwise the client will
+    // crash if it fails to execute the given handler executable.
```

Comment:
Technically, shell lookup rules apply. It can also be a relative path, but that would have to be prefixed with `./` to work in all cases.

Reply:
I updated the wording a bit, thanks